src/game.cpp

```diff
@@ -7141,6 +7141,16 @@ void game::smash()
             return; // don't smash terrain if we've smashed a corpse
         }
     }
+    bool ter_or_furn = m.has_flag_ter_or_furn( "ELECTRIFIED", smashx, smashy );
+    if (ter_or_furn) {
+            body_part bp = random_body_part();
```

Comment:
Placeholder damage for now. I could make it hurt both hands/arms, one arm randomly, chest and arms, or the whole body. Any other suggestions?

 I was not aware of conductivity with weapons. I'll add that.